WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -423,10 +418,10 @@ private async Task HandleCoinJoinFinalizationAsync(CoinJoinTracker finishedCoinJ
 				wallet.LogInfo($"{nameof(CoinJoinClient)} finished. Coinjoin transaction was not broadcast.");
 			}
 		}
-		catch (NoCoinsToMixException x)
+		catch (CoinJoinClientException clientException)
 		{
-			NotifyCoinJoinStartError(wallet, CoinjoinError.NoCoinsToMix);
-			Logger.LogDebug(x);
+			Logger.LogDebug(clientException);
```

Comment:
It was LogTrace before.

Reply:
No, it wasn't.
![image](https://user-images.githubusercontent.com/45069029/230117328-09a32b66-aeca-45c0-8c6d-5f1de228cbe7.png)

I can make it `LogTrace`, if we want, but it helped me while I was debugging.

Reply:
But all the others were.